docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,17 +1,17 @@
 [id="con_devfile_{context}"]
-= Overview of devfiles
+= About devfiles
 
 [role="_abstract"]
 
-.How devfiles work
+.How to use devfiles
 
-As a YAML file, devfiles simplify the development process. Devfiles tell you the steps you must take to get your application up and running from a source code to an actual product.When you download a devfile, it assesses your application development lifecycle, then provides you strategies and best practices for end-to-end application development. Devfiles do not force you to use new workflows nor tools, but rather, provide you instructions to automate your current, manual processes. For example, a devfile can take the repository hosting your application source code, build the code, run the application on your local container, then deploy the application to cloud-native containers. Unlike many development tools, devfiles can build, run, and deploy, automating many of your manual processes and simplifying your end-to-end development.
+Devfiles are YAML files, and they simplify your development process. In your IDE, devfiles tell you which best practices to apply to take your development project from source code to running application. Using the instructions given by devfiles, you can automate your current, manual processes. A devfile can take in the repository hosting your application source code. You can use the devfile to build your code, run your application on your local container, and then deploy your application to cloud-native containers.
```

Comment:
SMEs:

Doc reviewers had a good question on where devfiles give their suggestions: 
```I don't quite understand how devfiles do that. Do you open them in your terminal or in your IDE?
How does the devfile talk to the developer, is basically my question. How does it provide instructions?```

I think it's in the IDE. Am I correct? If not, where do users see devfiles' suggestions?

Reply:
It is the tools and IDEs that understand/support devfiles that do the work. Devfile captures instructions on how to do it so that all tools and IDEs behave the same when setting up the dev environment.

Reply:
Thanks, Elson, made the change.

Any SME reviewer, please see if I addressed this concern in the latest commit, thanks!